leaf-ui/js/displayAnalysis.js

```diff
@@ -330,6 +330,19 @@ function updateResults(){
     
 }
 
+/**
+ * Clear all results from all configs
+ */
+ function clearResults(){
+    for(let config of analysisMap.values()) {
+        // remove analysis results from each config
+        config.deleteResults();
+        // remove all results from config div
+        var dropdownElement = document.getElementById(config.id).querySelector('.dropdown-container');
```

Comment:
100% optional: You could use "$('.result-elements').remove();" here outside of the for-loop if you wanted to be (very slightly) more concise, but if you'd rather not use JQuery here understood

Reply:
Love this idea, I'll make that change!